### PR TITLE
fix: increase fresh period of f5 token to 18min

### PR DESCRIPTION
**Pull request by hanxueluo**
**What this PR does / why we need it**:

Add your description

**Which issue(s) this PR is related to** *(optional, link to 3rd issue(s))*:

Fixes #

Reference to #

**Special notes for your reviewer**:

/cc @hicaistar 

**Release note**:

```release-note
NONE
```

### CODE REVIEW

**Comment by hanxueluo** [comment]
规避问题：http://jira.caicloud.xyz/browse/INFINITUS-95

File: providers/f5lb/f5dns.go [hunk 1]
```diff
@@ -30,8 +30,8 @@ func refreshToken(c *f5CommonClient) error {
 	var err error
 
 	n := time.Now()
-	// default timeout of f5 token is 1200s, but we will refresh it in ervery 120s
-	if c.f5 == nil || time.Now().Add(time.Duration(-120)*time.Second).After(c.t) {
+	// default timeout of f5 token is 1200s, but we will refresh it in ervery 1080s(18min)
+	if c.f5 == nil || time.Now().Add(time.Duration(-1080)*time.Second).After(c.t) {
```
Comment (hicaistar) [inline providers/f5lb/f5dns.go:34]: 只是减少了失败次数，并没有彻底解决这个问题？

Comment (hanxueluo) [reply providers/f5lb/f5dns.go:34]: 是的。这只能算个优化，不是 solution。

**Pull request merged**